src/els_indexing.erl

```diff
@@ -215,3 +187,15 @@ index_dir(Dir, Mode) ->
              "[succeeded=~p] "
              "[failed=~p]", [Dir, Mode, Time/1000/1000, Succeeded, Failed]),
   {Succeeded, Failed}.
+
+-spec entries_apps() -> [{string(), 'deep' | 'shallow'}].
```

Comment:
Not requiring a change but why not make a single function out of these three that takes the name of the configuration entry as an argument?